src/components/Gameboard.ts

```diff
@@ -134,7 +134,7 @@ export class Gameboard {
 
     for (let row = areaStart[0]; row <= areaEnd[0]; ++row) {
```

Comment:
This is one instance where the notation change results in shadowing of function parameters, but since they're not used after being shadowed, there is no regression.